dev.yml

```diff
@@ -7,7 +7,7 @@ up:
     - openssl
     - sqlite
   - ruby:
-      version: 2.2.3p172-shopify
+      version: 2.3.3
```

Comment:
I couldn't dev up using `2.2.3p172-shopify` which is why I proposed this change. If it's going to cause issues I can work around that and close this PR